beacon-chain/sync/initial-sync/fsm_test.go

```diff
@@ -186,7 +186,7 @@ func TestStateMachine_trigger(t *testing.T) {
 				if err != nil && !expectHandlerError {
 					t.Error(err)
 				}
-				ind := uint64(tt.args.epoch) * 32
+				ind := types.Slot(tt.args.epoch * 32)
```

Comment:
`ind` is a slot?

Reply:
yes, state machines are indexed by start slot of the epoch.